ui/components/app/snaps/snap-privacy-warning/index.scss

```diff
@@ -1,15 +1,19 @@
 .snap-privacy-warning {
   &__content {
-    max-height: 325px;
     overflow-y: auto;
 
+    @include screen-sm-max {
+      max-height: 312px; // fits for popover view
+    }
+
+
     &__scroll-button {
-      position: absolute;
-      margin-left: auto;
-      margin-right: auto;
-      right: 0;
-      left: 0;
-      bottom: 110px;
+      position: sticky;
+      bottom: 16px;
+
+      @include screen-sm-min {
+        display: none; // hides scroll button
+      }
     }
   }
 }
```

Comment:
These styles make the modal easier to use for the different views so we are maxmizing realestate 

https://github.com/MetaMask/metamask-extension/assets/8112138/39dd5fff-32ad-41bf-a7e2-76eecb1bcffe